pkg/accountmanager/manager.go

```diff
@@ -102,11 +106,13 @@ func (a *AccountManager) AddAccount(namespacedName *types.NamespacedName, accoun
 	// Create an account poller for polling cloud inventory.
 	accPoller, exists := a.addAccountPoller(cloudInterface, namespacedName, account)
 	if !exists {
-		if !crd.DoesCesCrExistsForAccount(a.Client, namespacedName) {
+		numCes := crd.GetNumberOfCesCrForAccount(a.Client, namespacedName)
```

Comment:
GetCesCrForAccount Return CES and Count.. In the caller avoid using CES and just use count whereever possible